install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1440,6 +1440,12 @@ tls:
 #   - geneve
 tunnel: "vxlan"
 
+# -- Configure the VXLAN and Geneve tunnel port
+tunnelPort: ""
```

Comment:
I believe defaulting to a string here will break the config map validation, since this is expected to be an integer. Note that you can also leave helm values commented out like this:

https://github.com/cilium/cilium/blob/488dbae2c3392d299e08e310ced73085ff051199/install/kubernetes/cilium/values.yaml#L1923-L1924

Therefore I prefer to have those two value commented out by default (the Helm documentation will still be generated, even if commented out)

Reply:
@vincentmli This issue is still unresolved. While my comment was pointing out the tunnelPort, the issue also exists for the MTU variable.